qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/main.tf

```diff
@@ -50,7 +50,7 @@ resource "helm_release" "jupyterhub" {
               kind      = "configmap"
             }
 
-            "/opt/conda/envs/default/share/jupyter/lab/settings" = {
```

Comment:
I don't think this should be changed. This is the name of the conda env from the docker image.